drivers/gpio/gpio_stm32.c

```diff
@@ -250,9 +251,9 @@ int gpio_stm32_clock_request(const struct device *dev, bool on)
 
 #if defined(PWR_CR2_IOSV) && DT_NODE_HAS_STATUS(DT_NODELABEL(gpiog), okay)
 	z_stm32_hsem_lock(CFG_HW_RCC_SEMID, HSEM_LOCK_DEFAULT_RETRY);
+	/* Port G[15:2] requires external power supply */
```

Comment:
this modification could also come in the commit  "drivers/gpio: stm32: Don't perform clock gating with VddIO2 " 38c48f3af5bc47a2d2c31bdce731508677106951

Reply:
Fixed